native/src/main/java/org/wssec/WsSecurityUtils.java

```diff
@@ -0,0 +1,140 @@
+// Copyright (c) 2023, WSO2 LLC. (http://www.wso2.com) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
```

Comment:
```suggestion
// WSO2 LLC. licenses this file to you under the Apache License,
```
check all the places

Reply:
Fixed in [5eed3f5](https://github.com/ballerina-platform/module-ballerina-soap/pull/103/commits/5eed3f5c59d32aa0edc7e59e519bd4b671496ac0)